pkg/fanal/secret/scanner_test.go

```diff
@@ -609,6 +609,28 @@ func TestSecretScanner(t *testing.T) {
 			},
 		},
 	}
+	wantFindingHF := types.SecretFinding{
```

Comment:
You need to create a new test file + test case and use `wantFindingHF` as want result.

Reply:
Did this in https://github.com/aquasecurity/trivy/pull/6236/commits/758c4f5441f082ec421b9d5140130a971bfdde55